x-pack/plugins/security/public/components/management/users/edit_user.js

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/* eslint camelcase: 0 */
+import React, { Component, Fragment } from 'react';
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiTitle,
+  EuiForm,
+  EuiFormRow,
+  EuiIcon,
+  EuiText,
+  EuiFieldText,
+  EuiPage,
+  EuiComboBox,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageContentBody,
+  EuiHorizontalRule,
+  EuiSpacer,
+} from '@elastic/eui';
+import { toastNotifications } from 'ui/notify';
+import { USERS_PATH } from '../../../views/management/management_urls';
+import { ConfirmDelete } from './confirm_delete';
+
+export class EditUser extends Component {
```

Comment:
Can we add propTypes for this component, or convert it to TSX? It makes it a bit easier for me to understand how this component is consumed without needing to find a call site.